Gemfile

```diff
@@ -42,14 +42,14 @@ supports_windows = true
 
 group :development do
   gem 'puppet-lint',                        :require => false
-  gem 'metadata-json-lint',                 :require => false if ! supports_windows || Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.3.1')
+  gem 'metadata-json-lint',                 :require => false, :platforms => 'ruby'
```

Comment:
https://rubygems.org/gems/metadata-json-lint requires `json >= 0` ... so what's bundled in Ruby should already meet the requirement.

No need to exclude on Windows from what I see - but maybe there's something else you're seeing somewhere?